requirements/constraints.txt

```diff
@@ -191,39 +191,35 @@ google-api-core==1.8.0 \
     --hash=sha256:c6d834143a2bea4de8d1161b5460fd362457db40c55ea9ccbe672e5602e330af
 grpc-google-iam-v1==0.11.4 \
     --hash=sha256:5009e831dcec22f3ff00e89405249d6a838d1449a46ac8224907aa5b0e0b1aec
-grpcio==1.19.0 \
-    --hash=sha256:07c7f7b251b26ef94e29d2c19245e34d4d05897325a289b31de3b6a5e16fbd6c \
-    --hash=sha256:2ddbca16c2e7b3f2ffc6e34c7cfa6886fb01de9f156ad3f77b72ad652d632097 \
-    --hash=sha256:30d84f9684b4c81ee37906bb303a84435948c2dd3db55d3ef38f8daf28bc6ea3 \
-    --hash=sha256:316e6c79fb1585b23ae100ee26f6ffefa91a21e4d39588fa42efadd7f20c7225 \
-    --hash=sha256:400abff9a772351fff72d5698c8758b837bec3d7f4ed93de70bae744d8f63f53 \
-    --hash=sha256:4ed90a256f6f8690b5c95b9d4f2e9fe6513628f3674e9068e10637e50c2f93d6 \
-    --hash=sha256:51fd87ff610ca2f483c668c3fa7f70d479bffb3c14805d2065b51194edea5e26 \
-    --hash=sha256:5569aba69041530e04eff3d40536027db8851f4e11e6282849b9fc5b1855075d \
-    --hash=sha256:566b752e36cdcd5a4d38f292aca4c8e3095f13cfe82606e010d67749cacba341 \
-    --hash=sha256:5817f970fbfed72a6203ff96349e796d8f6ff3ce85b58af241c4a14190d9f4d1 \
-    --hash=sha256:5a97bb5a4af16f840f1211dbe66d61592f02110f286d96e67bf6006d7f96aab7 \
-    --hash=sha256:5d57e41c913152b215eda070955b3544bdf20ed2327e5e5eed3005186220ebd0 \
-    --hash=sha256:6cec17145978cef3d20093cdc05e88da597ce05076db566a66a35b9c55d416a3 \
-    --hash=sha256:6ef7ab9b6ba09ce087ddb3b27f12504f50efdbf5d319b8b23173478765452301 \
-    --hash=sha256:756c0d65e4ebce1c47787dbb48955864f2a768e1df76902f33d3e4062c209f3e \
-    --hash=sha256:828d13f0edd27f452af7fc23093c8a2d63d8fbd92595dbd0f698c78b13af9bdb \
-    --hash=sha256:8cf02c4e07520be61ad8b59b0043771ef2af666cb73066516eabfee562a28df4 \
-    --hash=sha256:919dfe84d22ce2e2ae81d82238586d7c2a86714fb0b6cf9b437e336851e3c32d \
-    --hash=sha256:b04a061280b06cdc4e68c4147a0f46b98c395cf62f0c6df4fa2a30a083cdc333 \
-    --hash=sha256:b2dbe7d2f9685bdbb4415f8e475dd96b1b1776193b7286705f90490c3f039037 \
-    --hash=sha256:b60df7cbc3e77c39d5befe6a1e6e4213f3ca683d743ff7c1622b1d4412245a55 \
-    --hash=sha256:b740681332b5a042b9e22246a3cdbfc3d644cf73d38e117f20ad9d8deab8f1a5 \
-    --hash=sha256:ba434873945d5d4542589674cb60c43a1cf76b2b5f0c0f759aa76d499055722f \
-    --hash=sha256:bcb44cd53beccc92c730254ad3d50715b67a7432e693961b566d982f759b1787 \
-    --hash=sha256:be1cbb6cad1d4242e3aaa4143eabcfbf383358f6c8e9951be2c497b65561b075 \
-    --hash=sha256:c4e38326fcab5c52fd1a8c8e0f908bfe830629a5ffc60793ec5545ef913d62d2 \
-    --hash=sha256:d03c0524d5953568f74269e0faebb1e880ba9f36ca8c773be397087c35bd8188 \
-    --hash=sha256:ea897ffa80276565acdd92349ef82a768db0e3327aacd4aec82f79ca10989689 \
-    --hash=sha256:edc50e8bcd10b165f34c3cf3e1d4f97e9c71b165b85a85b91cf3444000a17692 \
-    --hash=sha256:f96a2e97df522b50da9cb3795f08199b110ceab4146bf70ea7f6a3a0213786cc \
-    --hash=sha256:fadb649a69e3b08e01f090c24f0c8cccc122e92c362c1a1727b695a63be8416b \
-    --hash=sha256:fbe4360ff1689a9753cbf1b27dad11e683d39117a32a64372a7c95c6abc81b81
+grpcio==1.18.0 \
```

Comment:
1.19.0 causes segfaults now that we're (correctly) calling `init_gevent()`. Downgrading to 0.18.0 works fine.

I don't see anything in 1.19.0 fixes that might affect us, but who knows.